src/content/learn/manipulating-the-dom-with-refs.md

```diff
@@ -275,9 +275,14 @@ export default function CatFriends() {
 function setupCatList() {
   const catList = [];
   for (let i = 0; i < 10; i++) {
-    catList.push("https://loremflickr.com/320/240/cat?lock=" + i);
+    if (i < 5) {
+      catList.push("https://placecats.com/neo/320/240");
```

Comment:
We probably shouldn't break the key in the example. Instead, this can push an object like `{id: 1, src: 'https://...'}`, or you could add an index param to the URL like in the previous version (like `https://placecats.com/neo/320/240?id=1`).

Reply:
Looks like we do both options in other examples